engine/tester/config_parse.c

```diff
@@ -2094,12 +2190,40 @@ alloc_and_get_run_item(xmlNodePtr node, tester_cfg *cfg, unsigned int opts,
     {
         p->type = RUN_ITEM_SCRIPT;
         TAILQ_INIT(&p->u.script.reqs);
+        if (p->tmpl != NULL)
+            templ_script = p->tmpl->u.script.name;
         rc = get_script(node, cfg, &p->u.script);
         if (rc != 0)
             return rc;
+        if (~opts & TESTER_RUN_ITEM_TEMPLATE)
```

Comment:
I dislike that we have so much script specific code here. IMHO it is better to change get_script() signature and pass entire run_item p inside and update the function handle template semantics.